### ISSUE TITLE
Default marker function name conflicts

### ISSUE THREAD

**Issue by sinedied** [biesbjerg/ngx-translate-extract#55]
I'm trying to use marker function in my code, using aliasing to avoid conflicts:
```ts
import { _ as extract } from '@biesbjerg/ngx-translate-extract';
...
```
But it seems that aliasing or not, Karma does not like the fact that I there's a `_` function used here:
```
Error: Error encountered resolving symbol values statically. Calling function '_', function calls are not supported. Consider replacing the function or lambda with a reference to an exported function,
```

Maybe you should consider using a more explicit default marker function name, such as `extract` for example? This also cause issues when using popular libraries such as `lodash` or `underscore` in the project.

**Comment by PunkHaz4rd** [comment]
+1

**Comment by mohsenjavidpanah** [comment]
I also considered to this error in angular4(universal) project, i oped "xyz/node_modules/@biesbjerg/ngx-translate-extract/dist/utils/utils.d.ts" file and renamed '_' to other thing but this error continuesly rasing 😞

**Comment by EnzoEghermanne** [comment]
Support.  Please rename the `_` marker to `extract` or something more meaningful. `_` as a function name doesn't explain it's purpose in code and collides with other popular libraries.

**Comment by PunkHaz4rd** [comment]
Exactly, and if someone wants to use that, there's the `as` .

**Comment by bedag-moo** [comment]
You guys realize that there is nothing special about the _ function? If you'd rather call it extract you can simply declare such a function and tell ngx-translate-extract about it.

**Comment by sinedied** [comment]
> You guys realize that there is nothing special about the _ function? If you'd rather call it extract you can simply declare such a function and tell ngx-translate-extract about it.

That's not the issue, the problem is that the current naming cause conflict with other broadly used libraries and cause error during tests.

**Comment by bedag-moo** [comment]
This "problem" has a workaround that is shorter than your comment:

    function extract(...s: string) {}

and 

    ngx-translate-extract -m extract

**Comment by PunkHaz4rd** [comment]
You're actually missing a return s; for it to work.

**Comment by sinedied** [comment]
@bedag-moo That's what I already did, but still not the point of this issue.

**Comment by biesbjerg** [comment]
I've created a separate package for the marker function.

`npm install @biesbjerg/ngx-translate-extract-marker`

```ts
import { marker } from '@biesbjerg/ngx-translate-extract-marker';
```

**Comment by zavakare** [comment]
What version did this get fixed for? I am in 4.2.0 and it is not working
